dapps/assets/src/components/DigitalTwins.ts

```diff
@@ -0,0 +1,80 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+// vue imports
+import Component, { mixins } from 'vue-class-component';
+
+// evan.network imports
+import { EvanComponent } from '@evan.network/ui-vue-core';
+import { DigitalTwinService } from './DigitalTwinService';
+import { debounce } from 'lodash';
+
+@Component
+export default class DigitalTwinsComponent extends mixins(EvanComponent) {
+  twinService: DigitalTwinService = new DigitalTwinService();
+
+  isLoading = true;
+
+  data = [];
+
+  columns = [
+    { key: 'icon', label: '' },
+    'name',
+    'owner',
+    'updated',
+    'created',
+    { key: 'actions', label: '' }
+  ];
+
+  async mounted() {
+    await this.fetchInitial();
+  }
+
+  /**
+   * Watch if user scrolled down and load more twins when necessary
+   */
+  scrollHandler = debounce(
+    async function(ev) {
+      let bottomOfWindow =
+        ev.target.clientHeight + ev.target.scrollTop >= ev.target.scrollHeight;
```

Comment:
maybe add a little offset so the user doesn't needs to scroll hard to bottom, so we can start bit earlier loading, like 100pixels or so ... ?

Reply:
Set to 200px for now